src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx

```diff
@@ -12,6 +12,9 @@ import { useRichTextEditorContext } from '@components/RichTextEditor/context';
 const getCheckboxListStyles = (styles: Record<string, CSSProperties>, element: TElement): CSSProperties =>
     styles[element.children[0].textStyle as string];
 
+export const CHECKBOX_DIV_CLASSES = 'tw-flex tw-flex-row tw-pb-2 tw-gap-1.5 tw-items-center';
+export const CHECKBOX_SPAN_CLASSES = 'focus:tw-outline-none tw-w-auto tw-min-w-[10px]';
```

Comment:
Can we use this in this file?

Reply:
Yeah for sure, stupid me missed that one